tests/system/providers/amazon/aws/example_sagemaker.py

```diff
@@ -539,12 +564,14 @@ def delete_logs(env_id):
         train_model,
         await_training,
         create_model,
+        register_model,
         tune_model,
         await_tuning,
         test_model,
         await_transform,
         # TEST TEARDOWN
         delete_ecr_repository(test_setup["ecr_repository_name"]),
+        delete_model_group(test_setup["model_package_group_name"], register_model.output),
```

Comment:
What do we expect to be deleted in this phase when register_model raised exception (not the happy path)?